ingestion/src/metadata/ingestion/models/patch_request.py

```diff
@@ -134,7 +135,7 @@ class PatchedEntity(BaseModel):
     "fileFormats": True,
 }
 
-RESTRICT_UPDATE_LIST = ["description", "tags"]
+RESTRICT_UPDATE_LIST = ["description", "tags", "owner"]
```

Comment:
thanks, we were indeed missing this one. Reviewed the code logic, it's nice and clean.

The only thing I am missing here since this is maybe one of the most important pieces of the ingestion now, are some tests. Please include scenarios for:
- Adding new description & owner
- Not patching description/owner when it exists

We can add them here: https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/tests/integration/ometa/test_ometa_patch.py

Also, please let's take the chance to refactor that class to use https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/tests/integration/integration_base.py to generate the entity names & init the ometa client.

Finally, pls let's remove the sample data usage from the test. Better to have each test initialize a new entity with randomized names. This will ensure nothing clashes

Thanks 🙏

Reply:
Added the tests